appveyor.yml

```diff
@@ -1,10 +1,10 @@
 
 version: '{build}'
 
-image: Visual Studio 2017
+image: Visual Studio 2019
 
 environment:
-  QTDIR: C:\Qt\5.13\msvc2017_64
+  QTDIR: C:\Qt\5.13\msvc2019_64
```

Comment:
QT 5.13 does not provide binaries for msvc2019 however, msvc2017_64 is compatible.